content/en/docs/instrumentation/java/automatic/withspan.md

```diff
@@ -0,0 +1,125 @@
+---
+title: Using API and Annotations
+description: OpenTelemetry documentaion for using API and annotations with Java agent.
```

Comment:
I can't think of a good alternative right now, but we'll want something that captures this, basically:

* Use instrumentation annotations
* Annotate some spans sparingly ---> not going whole-hog on manual instrumentation

Any suggestions?

Reply:
Yeah, even I was not sure of the title. let's see if someone has a good suggestion, I'll update accordingly.